modulo3/Clase21/Practica2-puntos3y4resueltos.sql

```diff
@@ -0,0 +1,9 @@
+
+/* punto 3 */
+
+SELECT * FROM northwind.products  where ProductID = 5 ORDER BY ProductName;
+SELECT * FROM northwind.products  where CategoryID = 4 ORDER BY ProductName;
+
+/* punto 4 */
+
+SELECT * FROM northwind.products  where UnitsInStock >= 25 and UnitsInStock <= 40 ORDER BY ProductName;
```

Comment:
otra opcion seria SELECT * FROM northwind.products  where UnitsInStock between 25 and 40 ORDER BY ProductName;